### PR TITLE
rules: merge or/and/regular packages into single file

### PR DESCRIPTION
**Pull request by matheusalcantarazup**
This commit remove all or, and and regular packages that contains the
implementation of rules and merge them into a single file. This change
was made for for all rules for all languages that we support using
horusec-engine.

Signed-off-by: Matheus Alcantara <matheus.alcantara@zup.com.br>

**- What I did**

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

**Review by nathanmartinszup — APPROVED** [review]

**Review by iancardosozup — APPROVED** [review]

File: internal/services/engines/nginx/rules.go [hunk 1]
```diff
@@ -32,9 +36,73 @@ func extensions() []string {
...
-		not.NewNginxNotIncludeContentSecurityPolicyHeader(),
-		not.NewNginxNotIncludeServerTokensOff(),
+		NewIncludeXFrameOptionsHeader(),
+		NewIncludeXContentTypeOptionsHeader(),
+		NewIncludeContentSecurityPolicyHeader(),
+		NewIncludeServerTokensOff(),
+	}
+}
+
+func NewIncludeXFrameOptionsHeader() text.TextRule {
```
Comment (nathanmartinszup) [inline internal/services/engines/nginx/rules.go:46]: Why not divide like the others? `rules.go` and `role_manager.go`

Comment (matheusalcantarazup) [reply internal/services/engines/nginx/rules.go:46]: I think it's because it has fewer rules than others. Do you think this can be confusing?

Comment (nathanmartinszup) [reply internal/services/engines/nginx/rules.go:46]: As everyone tends to grow, in the future we would have to make these changes anyway.

Comment (matheusalcantarazup) [reply internal/services/engines/nginx/rules.go:46]: Make sense, I gonna split these rules too.

**Pull request merged**